src/main/java/edu/hw1/task1/Task1.java

```diff
@@ -0,0 +1,25 @@
+package edu.hw1.task1;
+
+import java.util.regex.Pattern;
+
+public class Task1 {
+
+    final String pattern = "[0-9]{2,}:[0-5]{1}[0-9]{1}";
```

Comment:
`BUG` давай сделаем это констатной. константы в java называются в upper snake case `ВОТ_ТАК`

```suggestion
    static final String PATTERN = "[0-9]{2,}:[0-5]{1}[0-9]{1}";
```